src/components/dar_collection_table/Actions.js

```diff
@@ -0,0 +1,118 @@
+const cancelGray = '#333F52';
+
+const hoverCancelButtonStyle = Styles.TABLE.TABLE_BUTTON_ICON_HOVER;
+const baseCancelButtonStyle = Object.assign(
+  {},
+  Styles.TABLE.TABLE_ICON_BUTTON,
+  {color: cancelGray},
+  { alignItems: 'center' }
+);
+
+export default function Actions(props) {
+  const { showConfirmationModal, collection, goToVote, consoleType, actions } = props;
+  const collectionId = collection.darCollectionId;
+
+  const openOnClick = async (collection) => {
+    showConfirmationModal(collection, 'open');
+  };
+
+  const cancelOnClick = (collection) => {
+    showConfirmationModal(collection, 'cancel');
+  };
+
+  const openButtonAttributes = {
+    keyProp: `${consoleType}-open-${collectionId}`,
+    label: 'Open',
+    isRendered: actions.includes('Open'),
+    onClick: () => openOnClick(collection),
+    baseColor: duosBlue,
+    hoverStyle: {
+      backgroundColor: duosBlue,
+      color: 'white'
+    },
+    additionalStyle: {
+      padding: '3% 7%',
+      fontSize: '1.45rem',
+      fontWeight: 600,
+      color: 'white',
+      marginRight: '8%'
+    }
+  };
+
+  const cancelButtonAttributes = {
+    keyProp: `${consoleType}-cancel-${collectionId}`,
+    isRendered: actions.includes('Cancel'),
+    onClick: () => cancelOnClick(collection),
+    style: baseCancelButtonStyle,
+    hoverStyle: hoverCancelButtonStyle,
+    dataTip: 'Cancel Elections',
+    icon: Block,
+  };
+
+  const voteButtonAttributes = {
+    keyProp: `${consoleType}-vote-${collectionId}`,
+    label: 'Vote',
+    isRendered: actions.includes('Vote'),
+    onClick: () => goToVote(collectionId),
+    baseColor: duosBlue,
+    hoverStyle: {
+      backgroundColor: duosBlue,
+      color: 'white',
+    },
+    additionalStyle: {
+      padding: '3% 7%',
+      fontSize: '1.45rem',
+      fontWeight: 600,
+      color: 'white',
+      marginRight: '8%',
+      border: `1px ${duosBlue} solid`,
+    },
+  };
+
+  const updateButtonAttributes = {
+    keyProp: `${consoleType}-update-${collectionId}`,
+    label: 'Update',
+    isRendered: actions.includes('Update'),
+    onClick: () => goToVote(collectionId),
+    baseColor: 'white',
+    hoverStyle: {
+      backgroundColor: 'white',
+      color: duosBlue
+    },
+    additionalStyle: {
+      padding: '3% 7%',
+      fontSize: '1.45rem',
+      fontWeight: 600,
+      color: duosBlue,
+      marginRight: '8%',
+      border: `1px ${duosBlue} solid`
+    }
+  };
+
+  return div(
+    {
+      className: '${consoleType}-actions',
+      key: `${consoleType}-actions-${collectionId}`,
+      id: `${consoleType}-actions-${collectionId}`,
+      style: {
+        display: 'flex',
+        padding: '10px 5px',
+        justifyContent: 'flex-start',
+        alignItems: 'center',
+      },
+    },
+    [
+      h(SimpleButton, openButtonAttributes),
+      h(SimpleButton, voteButtonAttributes),
+      h(SimpleButton, updateButtonAttributes),
+      h(TableIconButton, cancelButtonAttributes),
```

Comment:
for researchers, there is also: `Resume` `Review` and `Revise`: there were some backend changes needed to implement the `Resume` button. perhaps https://github.com/DataBiosphere/duos-ui/pull/1708 could be pulled in and then you could take in the changes added there to add the action here.

Reply:
Yeah at some point we will need to add those other options in. I was just working incrementally within this branch, since only chair and member are the only ones using the new summary.

I would have to look into the researcher actions component to understand it tho, since it's different! And I'd want to make sure I'm carrying any working bits here. For right now, I'd prefer to keep this branch targeted for just the chair actions then, and add in the researcher stuff after this has been approved! I'm afraid that converting the chair -> generic actions meant I deleted something important/necessary, and this is like a sanity check to make sure it's still going okay